gns3/main_window.py

```diff
@@ -1105,9 +1105,22 @@ def startupLoading(self):
                 sys.exit(1)
                 return
 
+        run_as_root_path = LocalConfig.instance().runAsRootPath()
+
         if not sys.platform.startswith("win") and os.geteuid() == 0:
+            # touches file to know that user has run GNS3 as root and to prevent
+            # from running as user
+            if not os.path.exists(run_as_root_path):
+                open(run_as_root_path, 'a').close()
+
             QtWidgets.QMessageBox.warning(self, "Root", "Running GNS3 as root is not recommended and could be dangerous")
 
+        if os.getuid() != 0 and os.path.exists(run_as_root_path):
```

Comment:
Shouldn't we use os.geteuid() instead of os.getuid() here? https://stackoverflow.com/questions/14950378/what-is-difference-between-os-getuid-and-os-geteuid